db/migrate/058_name_unnamed_redactions.rb

```diff
@@ -0,0 +1,19 @@
+class NameUnnamedRedactions < ActiveRecord::Migration[5.0]
+  def self.up
+  	# after titles are fixed we change the column so to not be null
+    change_column_null "redactions", "title", false
+    # gets each redaction
+    # sets redaction title equal to a string of the id if there is no title
+    Redaction.find_each do |redaction|
+      if redaction.title.empty?
+        redaction.title = redaction.id.to_s
```

Comment:
There's no need to use `.to_s` here, rails will take care of the conversion for you.